api/src/services/auth.service.ts

```diff
@@ -378,13 +381,19 @@ export class AuthService {
       data.email = token.email;
     }
 
-    user = await this.prisma.userAccounts.update({
+    await this.prisma.userAccounts.update({
       data,
       where: {
         id: user.id,
       },
     });
 
+    // Refetch user with full relations
+    user = await this.userService.findUserOrError(
+      { userId: user.id },
+      UserViews.full,
+    );
+
```

Comment:
some integration test was failing bc of this. The user needed to be return with all relations.